echo/adapter.go

```diff
@@ -17,6 +17,7 @@ import (
 // creates a proxy response object from the http.ResponseWriter
 type EchoLambda struct {
 	core.RequestAccessor
+	v2 core.RequestAccessorV2
```

Comment:
I think it would be better to separate V2 like the Gin adapter (since that is already in) - https://github.com/awslabs/aws-lambda-go-api-proxy/tree/master/gin

Reply:
@speza I used https://github.com/awslabs/aws-lambda-go-api-proxy/pull/100 as an example, which was also merged. But happy to refactor if that is the preferred way of adding V2 support?

Reply:
Oh, I didn't see that one! I think my personal preference would be separate. But it appears there is not any consistency!

Reply:
Seeing as Gin is one of the main framework examples for this project I think it makes sense to follow the same pattern, will refactor 👍

Reply:
Have refactored to match the pattern set in the Gin adapter